spec/result_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+RSpec.describe ServiceActor::Result do
+  it "defines a method ending with *? suffix for each attribute" do
+    result = described_class.new
+    result.name = "Sunny"
+
+    expect(result.respond_to?(:name?)).to eq true
+  end
+
+  context "String" do
```

Comment:
To follow the `RSpec/ContextWording` RuboCop convention, contexts should start with `when`, `with` or `without`.

```suggestion
  context "when input is a String" do
```